2024/02/23 17:53:35 [crit] 4531#4531: *11 connect() to unix:/home/run/gunicorn.sock failed (2: No such file or directory) while connecting to upstream, client: 95.168.164.194, server: 194.67.91.215, request: "GET /api/solve_instance?instance=B/\C HTTP/1.1", upstream: "http://unix:/home/run/gunicorn.sock:/api/solve_instance?instance=B/\C", host: "194.67.91.215"
2024/02/23 17:54:29 [crit] 4531#4531: *13 connect() to unix:/home/run/gunicorn.sock failed (2: No such file or directory) while connecting to upstream, client: 95.168.164.194, server: 194.67.91.215, request: "GET /api/solve_instance?instance=B/\C HTTP/1.1", upstream: "http://unix:/home/run/gunicorn.sock:/api/solve_instance?instance=B/\C", host: "194.67.91.215"
2024/02/23 17:59:20 [crit] 4787#4787: *1 connect() to unix:/home/run/gunicorn.sock failed (2: No such file or directory) while connecting to upstream, client: 95.168.164.194, server: 194.67.91.215, request: "GET /api/solve_instance?instance=B/\C HTTP/1.1", upstream: "http://unix:/home/run/gunicorn.sock:/api/solve_instance?instance=B/\C", host: "194.67.91.215"
2024/02/23 17:59:24 [crit] 4787#4787: *1 connect() to unix:/home/run/gunicorn.sock failed (2: No such file or directory) while connecting to upstream, client: 95.168.164.194, server: 194.67.91.215, request: "GET /api HTTP/1.1", upstream: "http://unix:/home/run/gunicorn.sock:/api", host: "194.67.91.215"
2024/02/23 17:59:28 [crit] 4787#4787: *1 connect() to unix:/home/run/gunicorn.sock failed (2: No such file or directory) while connecting to upstream, client: 95.168.164.194, server: 194.67.91.215, request: "GET /api/solve_instance?instance=B/\C HTTP/1.1", upstream: "http://unix:/home/run/gunicorn.sock:/api/solve_instance?instance=B/\C", host: "194.67.91.215"
2024/02/23 18:09:01 [crit] 4787#4787: *5 connect() to unix:/home/run/gunicorn.sock failed (2: No such file or directory) while connecting to upstream, client: 95.168.164.194, server: 194.67.91.215, request: "GET /api/solve_instance?instance=B/\C HTTP/1.1", upstream: "http://unix:/home/run/gunicorn.sock:/api/solve_instance?instance=B/\C", host: "194.67.91.215"
2024/02/23 18:11:40 [crit] 4787#4787: *8 connect() to unix:/home/run/gunicorn.sock failed (2: No such file or directory) while connecting to upstream, client: 95.168.164.194, server: 194.67.91.215, request: "GET /api/solve_instance?instance=B/\C HTTP/1.1", upstream: "http://unix:/home/run/gunicorn.sock:/api/solve_instance?instance=B/\C", host: "194.67.91.215"
2024/02/23 18:12:37 [crit] 4787#4787: *12 connect() to unix:/home/run/gunicorn.sock failed (2: No such file or directory) while connecting to upstream, client: 95.168.164.194, server: 194.67.91.215, request: "GET /api/solve_instance?instance=B/\C HTTP/1.1", upstream: "http://unix:/home/run/gunicorn.sock:/api/solve_instance?instance=B/\C", host: "194.67.91.215"
2024/02/23 18:14:20 [crit] 4787#4787: *14 connect() to unix:/home/run/gunicorn.sock failed (2: No such file or directory) while connecting to upstream, client: 95.168.164.194, server: 194.67.91.215, request: "GET /api/solve_instance?instance=B/\C HTTP/1.1", upstream: "http://unix:/home/run/gunicorn.sock:/api/solve_instance?instance=B/\C", host: "194.67.91.215"
2024/02/23 18:14:38 [crit] 4787#4787: *16 connect() to unix:/home/run/gunicorn.sock failed (2: No such file or directory) while connecting to upstream, client: 95.168.164.194, server: 194.67.91.215, request: "GET /api/solve_instance?instance=B/\C HTTP/1.1", upstream: "http://unix:/home/run/gunicorn.sock:/api/solve_instance?instance=B/\C", host: "194.67.91.215"
2024/02/23 18:15:24 [crit] 4787#4787: *18 connect() to unix:/home/run/gunicorn.sock failed (2: No such file or directory) while connecting to upstream, client: 95.168.164.194, server: 194.67.91.215, request: "GET /api/solve_instance?instance=B/\C HTTP/1.1", upstream: "http://unix:/home/run/gunicorn.sock:/api/solve_instance?instance=B/\C", host: "194.67.91.215"
